v4/export/config.go

```diff
@@ -70,6 +70,7 @@ const (
 	flagReadTimeout              = "read-timeout"
 	flagTransactionalConsistency = "transactional-consistency"
 	flagCompress                 = "compress"
+	flagSpeedLimit               = "speed-limit"
```

Comment:
Is this better? It seems that this argument can only affect write speed
```suggestion
	flagSpeedLimit               = "write-speed-limit"
```

Reply:
resolved.